src/DataMigration/DataMigration/Cmdlets/NewSsisMigrationInfo.cs

```diff
@@ -0,0 +1,56 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Commands.DataMigration.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.DataMigration.Models;
+
+namespace Microsoft.Azure.Commands.DataMigration.Cmdlets
+{
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "DataMigrationSsisMigrationInfo"), OutputType(typeof(MigrateSqlServerSqlDbDatabaseInput))]
```

Comment:
Must implement ShouldProcess

Reply:
Just creating a new object, don't think we need user confirmation.

Reply:
The build is failing because you don't implement shoudlprocess here.  Eitther you will need to implement it, or you will need to suppress the static analysis faiures.  Either is fine, but you will need to do one.

Reply:
Not really sure what you mean, ShouldProcess is a method of Cmdlet and not a virtual method. D o you mean something like Cmdlet([verbName], [nounName], SupportsShouldProcess = false)?

Reply:
We have a static analysis tool that makes sure every PR follows powershell convention -- but in this case, your `New-*` cmdlet doesn't change anything on Azure, so you don't have to implement `ShouldProcess`. However, you need to mark it as an exception so that static analysis tool won't block merging. Here's how: https://github.com/Azure/azure-powershell/blob/master/documentation/Debugging-StaticAnalysis-Errors.md#signature-issues